test/unit/io/GDALReaderTest.cpp

```diff
@@ -69,9 +70,9 @@ TEST(GDALReaderTest, simple)
     PointViewSet s = gr.execute(t);
     PointViewPtr v = *s.begin();
     PointLayoutPtr l = t.layout();
-    Dimension::Id id1 = l->findDim("band-1");
-    Dimension::Id id2 = l->findDim("band-2");
-    Dimension::Id id3 = l->findDim("band-3");
+    Dimension::Id id1 = l->findDim("Intensity");
```

Comment:
Probably good to have two tests -- one with header and one without.

Reply:
The other tests continue to use the "band-#" nomenclature.